jsonnet/components/kube-state-metrics.libsonnet

```
@@ -72,6 +72,34 @@ function(params)
       },
     },
 
+    kubeRbacProxySecret: {
+      apiVersion: 'v1',
+      kind: 'Secret',
+      metadata: {
+        name: 'kube-state-metrics-kube-rbac-proxy',
```

As @jan--f suggested, the secret could be more descriptive 

```suggestion
        name: 'kube-state-metrics-kube-rbac-proxy-config',
```

---

Done